src/components/Navbar.js

```diff
@@ -3,12 +3,23 @@ import { Link } from 'react-router-dom';
 
 function Navbar() {
   return (
-    <nav className="navClass">
-      <Link to="/">Books</Link>
-      <Link to="/categories">
-        Categories
-      </Link>
-    </nav>
+    <div className="navigation">
+      <div className="topnav__bar">
+        <div className="navlinks">
+          <Link to="/" className="logo__text">
+            Bookstore CMS
```

Comment:
- [ ] As requested by the project requirement, your style should fit exactly the original design provided by Zepplin. Kindly, make sure that your navbar looks as the navbar from the original design. Kindly, add the user icon as on the original design and also fix the gap between BOOKS and CATEGORIES menu items.

| Your Style | Original Style |
| ----------- | --------------- |
|![image](https://user-images.githubusercontent.com/71874425/173194590-a181a184-0b3e-406e-93ea-82ad5789f281.png)|![image](https://user-images.githubusercontent.com/71874425/173194501-a31453e7-11ad-497a-afb7-64c146e434fc.png)|